api/rosetta/options_integration_test.go

```diff
@@ -70,124 +70,118 @@ func TestAPI_Options(t *testing.T) {
 
 	assert.True(t, options.Allow.HistoricalBalanceLookup)
 
-	if assert.Len(t, options.Allow.OperationStatuses, 1) {
+	require.Len(t, options.Allow.OperationStatuses, 1)
 
-		status := options.Allow.OperationStatuses[0]
-		assert.Equal(t, status.Status, dps.StatusCompleted)
-		assert.True(t, status.Successful)
-	}
-
-	if assert.Len(t, options.Allow.OperationTypes, 1) {
-		assert.Equal(t, options.Allow.OperationTypes[0], dps.OperationTransfer)
-	}
+	status := options.Allow.OperationStatuses[0]
+	assert.Equal(t, status.Status, dps.StatusCompleted)
+	assert.True(t, status.Successful)
 
-	if assert.Len(t, options.Allow.Errors, wantErrorCount) {
+	require.Len(t, options.Allow.OperationTypes, 1)
+	assert.Equal(t, options.Allow.OperationTypes[0], dps.OperationTransfer)
 
-		for i := uint(0); i < wantErrorCount; i++ {
+	require.Len(t, options.Allow.Errors, wantErrorCount)
 
-			rosettaErr := options.Allow.Errors[i]
+	for i := uint(0); i < wantErrorCount; i++ {
+		rosettaErr := options.Allow.Errors[i]
 
-			expectedCode := i + 1 // error codes start from 1
+		expectedCode := i + 1 // error codes start from 1
+		assert.Equal(t, expectedCode, rosettaErr.Code)
 
-			assert.Equal(t, expectedCode, rosettaErr.Code)
+		switch expectedCode {
```

Comment:
Did you change the order here for a reason? Or was this broken by a rebase? There are now 23 error types we should check here, and the order is the same in which they were defined, so that makes it easier to keep it synced between configuration and tests.

Reply:
The order did not change, it's just the GitHub diff that is misleading. All that happened is that this was un-nested. LMK if I'm missing something but I don't think so.